tests/baselines/reference/api/typescript.d.ts

```diff
@@ -1964,7 +1964,7 @@ declare namespace ts {
         DiagnosticsPresent_OutputsSkipped = 1,
         DiagnosticsPresent_OutputsGenerated = 2,
         InvalidProject_OutputsSkipped = 3,
-        ProjectReferenceCycle_OutputsSkupped = 4
```

Comment:
you need to call `gulp runtests --t="Public API"` and  then `gulp baseline-accept`

Reply:
Done, properly.